spec/runner.js

```diff
@@ -0,0 +1,39 @@
+// Atom already has the capability to require("foo.ts") files, but it's using typescript-simple package at version 1.0,
+// that is instead using Typescript 1.4. We want to use whatever version is specified in out package.json, so we kick out
+// the typescript-simple handler and install our own via ts-node.
+var path = require("path")
+
+if (require.extensions[".ts"]) {
+  // Because <insert reason>, the ".ts" extension property is read only and not configurable, so we have to replace the
+  // whole extensions object to get rid of it.
+  var extensions = {}
+
+  for (var ext in require.extensions) {
```

Comment:
Can use `Object.assign({},require.extensions)` instead of this, and then `delete` from new object. A bit more compact, if nothing else, and I think `Object.assign` works somewhat faster on Chrome.